app/controllers/api/storage.php

```diff
@@ -245,28 +785,53 @@
     ->label('sdk.response.code', Response::STATUS_CODE_OK)
     ->label('sdk.response.type', Response::CONTENT_TYPE_JSON)
     ->label('sdk.response.model', Response::MODEL_FILE)
+    ->param('bucketId', null, new UID(), 'Storage bucket unique ID. You can create a new storage bucket using the Storage service [server integration](/docs/server/storage#createBucket).')
     ->param('fileId', '', new UID(), 'File ID.')
```

Comment:
Don't we have an upper limit on the number of characters for an ID ? Shouldn't we specify it as part of the UID() constructor ?

Reply:
It's already limited by the validator to 36 chars, we don't need to do anything

Reply:
Okay perfect 👍